core/src/main/scala/anorm/package.scala

```diff
@@ -32,6 +32,19 @@ package object anorm {
     }
   }
 
+  /** Structural type for Oracle timestamp wrapper. */
+  type TimestampWrapperOracle = { def timestampValue: java.sql.Timestamp }
```

Comment:
I would prefer that to be named more neutrally (e.g. `TimestampWrapper2`) even if the Scaladoc mentions it helps with Oracle, as it can be possibly be used for any other JDBC type providing `.timestampValue`.